cfme/tests/automate/test_domain.py

```diff
@@ -276,3 +276,40 @@ def test_domain_import_git(request, appliance, url, param_type, param_value, ver
     request.addfinalizer(domain.delete_if_exists)
     assert domain.exists
     domain.delete()
+
+
+@pytest.mark.tier(1)
+def test_object_attribute_type_in_automate_schedule(appliance):
+    """
+    Polarion:
+        assignee: ghubale
+        casecomponent: Automate
+        initialEstimate: 1/15h
+        startsin: 5.8
+        tags: automate
+        testSteps:
+            1. Go to Configuration > settings > schedules
+            2. Select 'Add a new schedule' from configuration drop down
+            3. selecting 'Automation Tasks' under Action.
+            4. Select a value from the drop down list of Object Attribute Type.
+            5. Undo the selection by selecting "<Choose>" from the drop down.
+        expectedResults:
+            1.
+            2.
+            3.
+            4. No pop-up window with Internal Server Error.
+            5. No pop-up window with Internal Server Error.
+
+    Bugzilla:
+         1479570, 1686762
+    """
+    view = navigate_to(appliance.collections.system_schedules, 'Add')
+    view.form.action_type.select_by_visible_text('Automation Tasks')
+    for options in view.form.object_type.all_options:
+        if options.text == "EVM Group" or options.text == "Tenant":
```

Comment:
Required: Why not use a BZ().blocks here for 1686762? This way you will be testing that option when the BZ is fixed. e.g. ```if options.text == "EVM Group" and BZ(1686762).blocks```